daprdocs/content/en/operations/monitoring/tracing/supported-tracing-backends/dynatrace.md

```diff
@@ -0,0 +1,83 @@
+---
+type: docs
+title: "How-To: Set up Dynatrace OneAgent for distributed tracing"
+linkTitle: "Dynatrace"
+weight: 1000
+description: "Set up Dynatrace OneAgent for distributed tracing and deep code-level insights in Kubernetes"
+---
+
+[Dynatrace](https://www.dynatrace.com) OneAgent is responsible for collecting all telemetry within your monitored environment. 
+
+It automatically instruments your applications built with Dapr without the need for further configuration. 
```

Comment:
Can you please explain in further detail how this happens? Does Dynatrace need to intercept the requests sent/received by the applications?

Currently, Dapr does that and then send trace to a Zipkin-supported URL, but it seems your sample didn't need to do that.